app/scripts/modules/core/src/pipeline/config/triggers/trigger.html

```diff
@@ -1,102 +1,111 @@
 <div class="row">
   <div class="col-md-12">
-    <div class="form-horizontal panel-pipeline-phase">
-      <div class="form-group row">
-        <div class="col-md-10">
-          <div class="row">
-            <label class="col-md-3 sm-label-right">Type</label>
-            <div class="col-md-9">
-              <select
-                class="input-sm"
-                ng-change="triggerCtrl.loadTrigger()"
-                ng-options="option.key as option.label for option in options"
-                ng-model="trigger.type"
-              >
-                <option value="">Select...</option>
-              </select>
-              {{description}}
+    <fieldset ng-disabled="trigger.inherited" ng-class="{'templated-pipeline-item': trigger.inherited}">
```

Comment:
Similar to Parameter.tsx changes, much of this is shifting elements and Prettier. Inheritance related changes are on lines 3, 22-32, 55